pom.xml

```diff
@@ -65,6 +64,7 @@
         <jdbc.postgresql-driver.version>9.4-1201-jdbc41</jdbc.postgresql-driver.version>
         <joda-time.version>2.3</joda-time.version>
         <log4j.version>1.2.17</log4j.version>
+        <lsp4j.version>0.2.0-SNAPSHOT</lsp4j.version>
```

Comment:
Snapshot is suitable for testing but not for merging in master. We might need to wait a release before merging